### PR TITLE
Individual axis homing menu items

### PR DESCRIPTION
**Pull request by jbrazio**
Adds a configurable option (disabled by default) which adds individual axis homing to the LCD menu. Idea taken from #3698.

### CODE REVIEW

File: Marlin/ultralcd.cpp [hunk 1]
```diff
@@ -1093,6 +1093,11 @@ static void lcd_prepare_menu() {
   // Auto Home
   //
   MENU_ITEM(gcode, MSG_AUTO_HOME, PSTR("G28"));
+  #if ENABLED(INDIVIDUAL_AXIS_HOMING_MENU)
+    MENU_ITEM(gcode, "Home X", PSTR("G28 X"));
+    MENU_ITEM(gcode, "Home Y", PSTR("G28 Y"));
+    MENU_ITEM(gcode, "Home Z", PSTR("G28 Z"));
+  #endif
```
Comment (thinkyhead) [inline Marlin/ultralcd.cpp]: I didn't show it in my original example, but these ought to be translatable strings.

Comment (jbrazio) [inline Marlin/ultralcd.cpp]: Do you believe we don't have a translation for the word "home" ? :-/

**Comment by jbrazio** [comment]
@thinkyhead changes applied.

File: Marlin/language_en.h [hunk 2]
```diff
@@ -64,6 +64,15 @@
...
 #endif
+#ifndef MSG_AUTO_HOME_X
+  #define MSG_HOME                            "Home X"
+#endif
+#ifndef MSG_AUTO_HOME_Y
+  #define MSG_HOME                            "Home Y"
+#endif
+#ifndef MSG_AUTO_HOME_Z
+  #define MSG_HOME                            "Home Z"
+#endif
```
Comment (thinkyhead) [inline Marlin/language_en.h]: Oops…

``` cpp
 #ifndef MSG_AUTO_HOME_X
   #define MSG_AUTO_HOME_X                     "Home X"
 #endif
 #ifndef MSG_AUTO_HOME_Y
   #define MSG_AUTO_HOME_Y                     "Home Y"
 #endif
 #ifndef MSG_AUTO_HOME_Z
   #define MSG_AUTO_HOME_Z                     "Home Z"
 #endif
```

**Comment by thinkyhead** [comment]
As options multiply it's getting very tempting to add a new `Configuration_LCD.h` file… Must resist…

File: Marlin/language_en.h (hunk 2 above)
Comment (jbrazio) [inline Marlin/language_en.h]: crap

**Comment by jbrazio** [comment]
@thinkyhead rebased.

**Comment by thinkyhead** [comment]
Squashed and soon to merge…

**Pull request closed without merging**